### PR TITLE
Added queueTransaction script and setting owner and platformFeeTo in deployGovernance

### PR DESCRIPTION
**Pull request by xenide**

### CODE REVIEW

File: scripts/queueTransaction.js [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+        console.error("Invalid network specified");
+        process.exit(1);
+    }
+}
+
+const web3 = thorify(new Web3(), rpcUrl);
+
+web3.eth.accounts.wallet.add(config.privateKey);
+
+queueTransaction = async() =>
```
Comment (OliverNChalk) [inline scripts/queueTransaction.js]: Should file / function be renamed to `queueTimelockTransfer`?

File: scripts/timelockChangeAdminAndGovernorAcceptAdmin.js [hunk 2]
```diff
@@ -69,36 +69,6 @@ timelockChangeAdminAndGovernorAcceptAdmin = async() =>
                         .admin().call() == changeAdminConfig.governorAlphaAddress);
 
         console.log("GovernorAlpha successfully accepted admin for Timelock");
```
Comment (OliverNChalk) [inline scripts/timelockChangeAdminAndGovernorAcceptAdmin.js:71]: Could rename this to `executeTimelockTransfer` for consistency?

**Review by OliverNChalk — COMMENTED** [review]
Changes look good, suggested a potential name change - let me know what you think

File: scripts/queueTransaction.js (hunk 1 above)
Comment (xenide) [reply scripts/queueTransaction.js]: Yes makes sense, changed both names.

**Review by OliverNChalk — APPROVED** [review]

**Comment by OliverNChalk** [comment]
Looks good, please squash and merge once you're happy

**Review by leodsoft — APPROVED** [review]
yep looks good

**Pull request merged**